lib/bridge/github/issues.rb

```diff
@@ -16,9 +16,9 @@ def issues(label = nil, opts={})
 
       issues_response = gh.issues(params) do |request|
         request.headers["Accept"] = "application/vnd.github.squirrel-girl-preview"
-      end
+      end.all
 
-      issues_response.all.each{
+      issues_response.each{
```

Comment:
I'm not sure, Ghee does some weird magically stuff sometimes. 

Might be a quirk in Ghee.

I think it's because you have a pointer reference to the proxy object in the old diff, the proxy object must not modify itself or consider itself "fulfilled" so subsequent calls to all would execute again? 

Not sure why that would cause double entries in the array tho